cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -138,7 +138,7 @@ install_docker(){
 			sudo -E apt-get -y install "${pkg_name}=${docker_version_full}"
 		elif [[ "$ID" =~ ^opensuse.*$ ]] || [ "$ID" == "sles" ]; then
 			sudo zypper removelock docker
-			sudo zypper -n  install 'docker<18.09'
+			sudo zypper -n  install 'docker<19.03'
```

Comment:
the change probably looks good, but it is a bit confusing with the commit message - from the commit message I was expecting to see 18.06->18.09, but we have 18.09->19.03 ... can you clarify that for me a bit?

Reply:
@grahamwhaley this will install docker 18.09, this is similar to do zypper in docker-18.09.6_ce

Reply:
@grahamwhaley 
```
Ralf Haferkamp  11:17 AM
To install a specific older version you can do zypper in docker-18.09.1_ce or you can to something like zypper in "docker<19.03"
```